content/blog/2021-07-16-july-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,247 @@
+﻿---
+title: July '21 Heartbeat
+date: 2021-07-16
+description: |
+  Monthly updates are here! Great new tutorials from the Community, uptick in 
+  jobs requiring DVC, awesome Community discussion on experiments at our June
+  Meetup and a cipher. Can you figure it out?
+descriptionLong: |
+  This month you will find:
+  📈 DVC + Streamlit = ❤️,
```

Comment:
was the intention to render it like a list? I think it's supported, you probably need to do it a proper Markdown list though

Reply:
Ok, next push up will have a list, but has bullet points.  I would rather the emojis **be** the bullet points but I can't figure out how to do it.  I found this and tested it, but it didn't work: https://stackoverflow.com/questions/60131406/replace-bullet-points-with-emojis-or-other-unicode-characters-but-dont-chang

Reply:
>  I would rather the emojis be the bullet points 

yep, but also don't know an easy way to do this. 

My 2cs - list (even with ugly points + emojis) looks still better. But it's totally your call on this :)

Reply:
I'm good with it!